_posts/2018-03-21-peripheral-management.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Safe Peripheral Management
+authors: ppannuto
+
+---
+
+[pr]: https://github.com/helena-project/tock/pull/760
+[obrm]: https://doc.rust-lang.org/beta/nomicon/obrm.html
+[usart-commit]: https://github.com/helena-project/tock/pull/760/commits/d975cd321345bc40e9fdf471d9a82fcd0db84b55#diff-001c67ee9f8c56d6403b71536b6ebecbR290
+[usart-int]: https://github.com/helena-project/tock/pull/760/commits/d975cd321345bc40e9fdf471d9a82fcd0db84b55#diff-001c67ee9f8c56d6403b71536b6ebecbL528
+[mmio]: https://en.wikipedia.org/wiki/Memory-mapped_I/O
+[tt34]: https://www.tockos.org/blog/2018/talking-tock-34/
```

Comment:
Change to `https://www.tockos.org/blog/2018/talking-tock-34/#register-and-bitfield-macros`